peer/peerlist/list.go

```diff
@@ -66,6 +72,24 @@ func Capacity(capacity int) ListOption {
 	})
 }
 
+// Shuffle enables/disables shuffling the order of peers as they are added.
+//
+// Default is true.
+func Shuffle(shuffle bool) ListOption {
```

Comment:
If the default is true, we just just make this a `DisableShuffling()` option instead of taking a boolean.  Makes it easier to reason about what is happening.

Reply:
changed option signature to be `func NoShuffle() ListOption`, with no argument.